server/common/utils/index.test.js

```diff
@@ -0,0 +1,391 @@
+  responseWithCohortMember,
+  responseWithCohortMembers,
+  responseWithCohorts,
+  responseWithItem,
+  responseWithItems,
+  responseWithList,
+  responseWithListMember,
+  responseWithListMembers,
+  responseWithLists
+} = require('./index');
+const {
+  expectedCohortListProperties,
+  expectedListMetaDataProperties,
+  expectedListProperties,
+  listMock
+} = require('../../tests/__mocks__/listMock');
+const { expectedItemProperties } = require('../../tests/__mocks__/itemsMock');
+const { singleItemMock } = require('../../tests/__mocks__/singleItemMock');
+const {
+  cohortsMock,
+  expectedCohortMetaDataProperties
+} = require('../../tests/__mocks__/cohortsMock');
+const {
+  expectedUsersProperties,
+  usersMock
+} = require('../../tests/__mocks__/usersMock');
+
+describe('function isValidMongoId', () => {
+  it('should return true', () => {
+    const id = ObjectId();
+    const result = isValidMongoId(id);
+
+    expect(id).toBeInstanceOf(ObjectId);
+    expect(result).toBe(true);
+  });
+
+  it('should return false', () => {
+    const id = '1243sb3';
+    const result = isValidMongoId(id);
+
+    expect(result).toBe(false);
+  });
+});
+
+describe('function isUserFavourite', () => {
+  it('should return true', () => {
+    const favIds = ['123', '456', '789'];
+    const userId = '123';
+
+    const result = isUserFavourite(favIds, userId);
+    expect(result).toBe(true);
+  });
+
+  it('should return true', () => {
+    const newId = ObjectId();
+    const favIds = ['123', '456', '789'];
+
+    favIds.push(newId);
+
+    const result = isUserFavourite(favIds, newId);
+    expect(result).toBe(true);
+  });
+
+  it('should return false', () => {
+    const userId = '999';
+    const favIds = ['123', '456', '789'];
+
+    const result = isUserFavourite(favIds, userId);
+    expect(result).toBe(false);
+  });
+
+  it('should return false', () => {
+    const userId = '999';
+    const favIds = [];
+
+    const result = isUserFavourite(favIds, userId);
+    expect(result).toBe(false);
+  });
+});
+
+describe('function responseWithList', () => {
+  const list = listMock[0];
+  const userId = ObjectId();
+
+  list.cohortId = ObjectId();
+
+  const result = responseWithList(list, userId);
+
+  it('should return object with desired properties', () => {
+    expectedListProperties.map(property =>
+      expect(result).toHaveProperty(property)
+    );
+  });
+
+  it('should return object without favIds property', () => {
+    const notExpected = 'favIds';
+
+    expect(result).not.toHaveProperty(notExpected);
+  });
+
+  it('should return object with desired properties for cohort list', () => {
+    expectedCohortListProperties.map(property =>
+      expect(result).toHaveProperty(property)
+    );
+  });
+});
+
+describe('function responseWithLists', () => {
+  const userId = ObjectId();
+  const lists = responseWithLists(listMock, userId);
+
+  it('expect to have only desired meta-data properties without sensitive data', () => {
```

Comment:
if so then maybe `responseWithLists` is not the right name? Maybe it should be `reponseWithListMetaData`?